rocky/tests/test_landing_page.py

```diff
@@ -7,3 +13,18 @@ def test_landing_page_redirect(rf, client_member):
 
     response = LandingPageView.as_view()(request)
     assert response.status_code == 302  # Redirects to crisis-room
+
+
+LANG_LIST = [lang[0] for lang in settings.LANGUAGES]
```

Comment:
Since it's a tuple with ([language code](https://docs.djangoproject.com/en/4.2/topics/i18n/#term-language-code), language name)
```suggestion
LANG_LIST = [code for code, _ in settings.LANGUAGES]
```